portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,524 @@
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body onload="getAnnouncements()">
+
+    <nav class="navbar fixed-top navbar-expand-lg navbar-dark scrolling-navbar top-nav-collapse">
+      
+      <div class="container">
+
+        <a class="navbar-brand" href="#">
+          <img class="avatar" src="img/avatar.png" style="width: 40px!important;height: auto;" alt="">
+          <strong>Arjan Singh Bal</strong>
+        </a>
+
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02"
+          aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+
+          <span class="navbar-toggler-icon"></span>
+
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+
+          <ul class="navbar-nav mr-auto smooth-scroll">
+
+            <li class="nav-item">
+              <a class="nav-link" href="#about" data-offset="100">About</a>
+            </li>
+
+            <li class="nav-item">
+              <a class="nav-link" href="#experience" data-offset="100">Experience </a>
+            </li>
+
+            <li class="nav-item">
+              <a class="nav-link" href="#cp" data-offset="100">Competitve Programming </a>
+            </li>
+
+            <li class="nav-item">
+              <a class="nav-link" href="#projects" data-offset="100">Projects  </a>
+            </li>
+
+            <li class="nav-item">
+              <a class="nav-link" href="#contact" data-offset="100">Contact  </a>
+            </li>
+
+          </ul>
+        </div>
+
+      </div>
+      
+    </nav>
+    
+    <div id="slides" class="carousel slide carousel-fade" data-ride="carousel">
+      <div class="carousel-inner">
+
+        <div class="carousel-item active">
+          <img class="d-block w-100" src="img/carousal/01.jpg">
+        </div>
+
+        <div class="carousel-item">
+          <img class="d-block w-100" src="img/carousal/03.jpg">
+        </div>
+
+        <div class="carousel-item">
+          <img class="d-block w-100" src="img/carousal/04.jpg">
+        </div>
+
+        <div class="carousel-item">
+          <img class="d-block w-100" src="img/carousal/06.jpg">
+        </div>
+
+        <div class="carousel-item">
+          <img class="d-block w-100" src="img/carousal/07.jpg">
+        </div>
+        
+      </div>
+
+      <div class="carousel-content text-center white-text wow fadeIn">
+        <div class=" row mx-0 headfont mb-5 pb-3">
+          <div class="col-12 col-sm-5 align-middle">
+            <img class="pull-right avatar avatar-lg ml-4 rounded-circle " src="img/arjan.jfif" alt="" >
+          </div>
+
+          <div class="col-12 col-sm-7 text-left pl-5">
+            <h1 class="display-4 mb-2 mt-2 h1">
+              <strong>Arjan Singh Bal</strong>
+            </h1>
+
+            <p class="mb-0 p" style="font-size: 1.250rem; font-weight: 200;">
```

Comment:
Never use inline css/
https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css

Reply:
Got it, removed inline styles